### PR TITLE
Display GH stars with projects

### PR DESCRIPTION
**Pull request by iankronquist**
Needs a little bit of CSS help
@tschuy r?

### CODE REVIEW

File: migrations/20141229012503_00.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ exports.up = function(knex, Promise) {
     table.string('hn_url');
     table.string('gh_url').primary();
     table.string('gh_name');
+    table.integer('gh_stars');
```
Comment (tschuy) [inline migrations/20141229012503_00.js]: We can't edit the existing migrations since they're already deployed :(
Please make a new migration.

**Comment by iankronquist** [comment]
Alright, I won't be so lazy then.

**Comment by iankronquist** [comment]
@tschuy You're going to need to drop the DB to repopulate it with stars data anyway. Nonetheless, here is the new migration.

**Comment by tschuy** [comment]
I'll just manually add the column and give it a default value then.

**Comment by tschuy** [comment]
```
.gh-stars {
  white-space: nowrap;
}
```

Looks good to me save that CSS. You can just nowrap it to make sure it's always on one line.

**Comment by iankronquist** [comment]
Fixed CSS and force pushed.

**Comment by tschuy** [comment]
+1 looks good to me, gimme a sec to add the column and then you can merge

File: migrations/20160123121933_🌟stars🌟.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table('ghprojects', function (table) {
+    table.integer('gh_stars').primary();
```
Comment (tschuy) [inline migrations/20160123121933_🌟stars🌟.js]: woah wait why is this primary

**Comment by tschuy** [comment]
Sweet, merging

**Pull request merged**